common/src/main/scala/gu/msnotifications/Responses.scala

```diff
@@ -65,7 +66,11 @@ object RegistrationResponse {
   }
 }
 case class RegistrationResponse(registration: WNSRegistrationId, tags: List[String], channelUri: String, expirationTime: DateTime) {
-  def toRegistrarResponse : RegistrarResponse = ???
+  def toRegistrarResponse : RegistrarResponse = RegistrarResponse(
+    deviceId = channelUri,
+    WindowsMobile,
+    userId = Tags(tags.toSet).findUserId.get
```

Comment:
I didn't want to make it Optional - RegistrationResponse without userId tag inside is considered as serious problem and I want to throw Exception here if you don't mind.